src/components/VDisk/VDisk.tsx

```diff
@@ -46,7 +46,7 @@ export const VDisk = ({
             onShowPopup={onShowPopup}
             onHidePopup={onHidePopup}
             renderPopupContent={() => <VDiskPopup data={data} />}
-            offset={{mainAxis: 5, crossAxis: 0}}
+            offset={{mainAxis: 2, crossAxis: 0}}
```

Comment:
Please, explain, how your changes make everything better? I see no difference

Reply:
The main difference is that YDB_POPOVER_CLASS_NAME is now applied not to the popup itself, but to its content. Previously, when paddings were set at the popup level, the popup would close when the mouse hovered over these paddings. Now, paddings are part of the content, so the popup no longer closes prematurely.
I also made some minor adjustments to the offsets, but that's not the main improvement.